HTML-CSS/homework.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <title>Shuaibu's HTML homework</title>
+</head>
+<body>
+    <div class="divClass">
+        <h1>Shuaibu Muhammad</h1>
+        <h3>Paritie Precourse</h3>
+        <h4>HTML/CSS homework</h4>
+    </div>
+    <div class="divClass">
+        <span id="spanId">My favourite food is pounded yam and melon soup, Pounded yam is belong to the class of food known as carbonhydrate</span>
+        <a href="#" title="Africook.com">Here is a link to my favourite Resturant</a>
```

Comment:
the link to the restaurant should be placed in the href tag while an appropriate title is included in the title tag